.github/workflows/_extension_distribution.yml

```diff
@@ -56,7 +56,7 @@ jobs:
 
       # CI tools is pinned to main
       override_ci_tools_repository: duckdb/extension-ci-tools
-      ci_tools_version: main
+      ci_tools_version: v1.4.3
```

Comment:
I believe the version also needs to be changed on line 43, when the workflow is imported.